libcst/_nodes/tests/test_import.py

```diff
@@ -564,6 +578,25 @@ def test_valid(self, **kwargs: Any) -> None:
                 ),
                 "expected_re": "one space after import",
             },
+            {
+                "get_node": lambda: cst.ImportFrom(
+                    module=cst.Name("foo"),
+                    names=(
+                        cst.ImportAlias(
+                            cst.Name("bar"),
+                            asname=cst.AsName(
+                                cst.Name(
+                                    "baz",
+                                    lpar=(cst.LeftParen(),),
+                                    rpar=(cst.RightParen(),),
+                                ),
+                                whitespace_before_as=cst.SimpleWhitespace(""),
+                            ),
+                        )
```

Comment:
Found the type check problem - need a trailing comma to make `name` a tuple:
```suggestion
                        ),
```

I'd be tempted to just use lists instead, tuples are confusing because of the trailing-comma-for-singleton issue. But if we switch to lists we'd want to do it everywhere in a dedicated PR.